packages/gestalt/src/ExpandableModule.css

```diff
@@ -0,0 +1,5 @@
+.expandablemodule {
+  background: rgba(0, 0, 0, 0);
```

Comment:
@galaxy0101 Could we remove this line? It basically adds a transparent background

Reply:
Removed